task_10/Oleg.Mikhailov/src/main/java/com/hillel/tasks/StringTasks.java

```diff
@@ -0,0 +1,67 @@
+package com.hillel.tasks;
+
+import java.util.Arrays;
+
+public class StringTasks {
+
+  public String replaceEmail(String email) {
+    String result = email.replace("@", "[ at ]");
+
+    result = result.replace(".", "[ dot ]");
+
+    return result;
+  }
+
+  public String shortenWords(String sentence) {
+    int countSymbols = 0;
+    int size = 0;
+    String[] arrayString = sentence.trim().split(" ");
+    StringBuilder result = new StringBuilder();
```

Comment:
Use StringJoiner here, you can add separator as
StringJoiner joiner = new StringJoiner(" ");